Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/Networking/Forge/Networking/NetWorker.cs

```diff
@@ -617,10 +617,11 @@ public bool RegisterNetworkObject(NetworkObject networkObject, uint forceId = 0)
 					id = forceId;
 
 					if (!networkObject.RegisterOnce(id))
+					{
 						throw new BaseNetworkException("The supplied network object has already been assigned to a networker and has an id");
+					}
```

Comment:
This should not be between `{` and `}` as per the contributing guidelines.

Reply:
Do you mean that there should be no brackets since it's one line code?

Reply:
yup

Reply:
just removed those brackets :)